### DISCUSSION TITLE
"where" operator help with XML/UQL

### DISCUSSION THREAD

**Discussion by jordan-xlab** [grafana/grafana-infinity-datasource#352] [Q&A] [answered]
The "where" operator seems to be supported (https://github.com/yesoreyeram/uql/blob/main/src/parser/where/where.test.ts), but I am struggling to get it working with XML & UQL.

![image](https://user-images.githubusercontent.com/83219602/177752877-7b64a870-461c-4020-ad6d-8ce69a48e878.png)

Keep getting an 'error applying uql query' response when I use this. As filters are not available when using UQL, I was hoping I could filter in the query.

Am I doing something wrong here? Is my syntax incorrect?

**Comment by yesoreyeram** [comment, marked as answer]
It is available from v1.0.0 https://github.com/yesoreyeram/grafana-infinity-datasource/releases/tag/v1.0.0-dev.5 and it is pre-release

**Reply by jordan-xlab** [reply]
thank you!
